jvm/src/main/kotlin/app/cash/trifle/internal/signers/TrifleContentSigner.kt

```diff
@@ -1,8 +1,9 @@
 package app.cash.trifle.internal.signers
 
+import com.google.common.annotations.VisibleForTesting
 import org.bouncycastle.asn1.x509.SubjectPublicKeyInfo
 import org.bouncycastle.operator.ContentSigner
 
-internal sealed class TrifleContentSigner : ContentSigner {
+internal abstract class TrifleContentSigner : ContentSigner {
```

Comment:
lets make this an interface instead and update the relevant implementing classes

Reply:
done